src/main/java/com/TeachMeSkills/jsp/hw/servlets/RegisteredServlet.java

```diff
@@ -0,0 +1,36 @@
+package com.TeachMeSkills.jsp.hw.servlets;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+@WebServlet("/registration")
+public class RegisteredServlet extends HttpServlet {
+
+    @Override
+    protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+        String firstName = request.getParameter("firstName");
+        String lastName = request.getParameter("lastName");
+        String amount = request.getParameter("amount");
+        if (!isAnyValueNullOrEmpty(firstName, lastName, amount)) {
+            request.setAttribute("firstName", firstName);
+            request.setAttribute("lastName", lastName);
+            request.setAttribute("amount", amount);
+            getServletContext().getRequestDispatcher("/WEB-INF/registered.jsp").forward(request, response);
+        } else {
+            getServletContext().getRequestDispatcher("/WEB-INF/save-request.jsp").forward(request, response);
+        }
+    }
+
+    private boolean isAnyValueNullOrEmpty(String firstName, String lastName, String amount) {
+        return firstName.isEmpty()
```

Comment:
It seems, this method could produce NPE

Reply:
Updated: [e0c3ca42](https://github.com/Alexander-Khamitski/TMS_Java_EE/pull/4/commits/e0c3ca425f7c656b848e46d68534e274602c8430)